### PR TITLE
Edited aggregation pipeline doc, added that you can adjust stage width

### PR DESCRIPTION
**Pull request by JuliaMongo**
[Jira](https://jira.mongodb.org/browse/DOCSP-15305)
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=608a18cd09293ae626b0d4a5) has no issues.

Made these changes:
- Edits, to clean up the existing content for [Aggregation Pipeline Builder](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-15305/aggregation-pipeline-builder/) and make it adhere to style guide by using the [Vale linter](https://github.com/10gen/mongodb-vale/blob/main/README.md).
- Added a line about changing width of the stage, in [Step 2](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-15305/aggregation-pipeline-builder/#fill-in-the-pipeline-stage) of this procedure.

Staging 1:  [Aggregation Pipeline Builder](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-15305/aggregation-pipeline-builder/)
Staging 2: [Step 2](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-15305/aggregation-pipeline-builder/#fill-in-the-pipeline-stage)
Thank you for your review!

### CODE REVIEW

File: source/includes/steps-create-agg-pipeline.yaml [hunk 1]
```diff
@@ -16,10 +16,11 @@ title: Fill in the pipeline stage.
 level: 4
 ref: fill-in-stage
 content: |
-   Fill in your selected stage. As you modify the pipeline stage, the
-   preview documents shown in the pane to the right of the stage
-   update automatically to reflect the results of your pipeline as it
-   progresses, provided :guilabel:`Auto Preview` is enabled:
+   Fill in your selected stage. If you enable :guilabel:`Auto Preview`,
```
Comment (jmd-mongo) [inline source/includes/steps-create-agg-pipeline.yaml:19]: [suggestion]

Consider: "If you enable :guilabel:`Auto Preview`, preview documents will update automatically to the right of the stage to reflect the results of your pipeline as it progresses."

**Review by jmd-mongo — COMMENTED** [review]
Overall LGTM, just left one small suggestion. Thanks!

File: source/includes/steps-create-agg-pipeline.yaml (hunk 1 above)
Comment (JuliaMongo) [reply source/includes/steps-create-agg-pipeline.yaml:19]: Done

**Review by jmd-mongo — APPROVED** [review]
LGTM! Thanks!

File: source/aggregation-pipeline-builder.txt [hunk 2]
```diff
@@ -14,65 +14,59 @@ Aggregation Pipeline Builder
...
-are presented with a blank aggregation pipeline. The
-:guilabel:`Preview of Documents in the Collection` section of the
-:guilabel:`Aggregations` view displays 20 documents sampled from the
-current collection.
+To start building an aggregation pipeline for a :doc:`collection or view
+</collections>`, choose the collection and click the
+:guilabel:`Aggregations` tab.
+A blank aggregation pipeline displays. The :guilabel:`Preview of
+Documents in the Collection` section of the :guilabel:`Aggregations`
+view displays 20 documents sampled from the current collection.
```
Comment (Anemy) [inline source/aggregation-pipeline-builder.txt:28]: The 20 documents we show is a configurable number. Maybe we can show that somehow:
`view displays 20 (configurable amount) documents sampled`

I'm not sure if it's just for me, but I think the previous wording for the blank aggregation pipeline read a little more smoothly:
`You are presented with a blank aggregation pipeline.`
vs
`A blank aggregation pipeline displays.`

Maybe we can say `A blank aggregation pipeline is displayed.`
All fine by my me though, just a couple ideas.

**Review by Anemy — APPROVED** [review]
lgtm! one idea, but looks good.

**Pull request merged**